scene/animation/tween.h

```diff
@@ -302,4 +307,26 @@ class MethodTweener : public Tweener {
 	Ref<RefCounted> ref_copy;
 };
 
+class SubtweenTweener : public Tweener {
+	GDCLASS(SubtweenTweener, Tweener);
+
+	friend class Tween;
```

Comment:
?

Reply:
This is because `SubtweenTweener::subtween` is a private member, but the `Tween::tween_subtween` method needs to be able to access it.

I see a few possibilities for how to handle this:
- Keep `friend class Tween` and `subtween` as `private`
- Remove `friend class Tween`, make `subtween` be `public`
- Remove `friend class Tween`, keep `subtween` as `private` but add a `SubtweenTweener::get_subtween` method so that the subtween can be *retrieved* from the `SubtweenTweener`, but the variable itself cannot be modified from outside the class instance

Which of these do you think would be best?